docker/helm/templates/kyuubi-deployment.yaml

```diff
@@ -36,6 +36,8 @@ spec:
       labels:
         app.kubernetes.io/name: {{ .Chart.Name }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+        checksum/conf: {{ include (print $.Template.BasePath "/kyuubi-configmap.yaml") . | sha256sum }}
```

Comment:
I'm not really familiar w/ helm, is it recommended? where can I find docs on that?

Reply:
Yes, it's official helm recommended [Chart Development Tips and Tricks](https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments) which actually sounds funny.
It's used in Apache Airflow [helm chart](https://github.com/apache/airflow/blob/672264b0af4874274bd130ba42a78e8e72f3d3ff/chart/templates/webserver/webserver-deployment.yaml#L79-L86) for example.